dags/multipod/configs/maxtext_sweep_gke_config.py

```diff
@@ -54,16 +59,22 @@ def get_maxtext_sweep_gke_config(
 
   # Generate all combinations of sweep param configurations and create a TpuXpkTask for each one
   xpk_task_list = []
-  current_datetime = datetime.datetime.now().strftime("%Y-%m-%d-%H-%M-%S")
   for idx, config in enumerate(itertools.product(*sweep_params_list)):
     config_dict = {key: value for (key, value) in config}
 
     # Remove num_slices as a sweep param after combinations have been generated
     curr_num_slices = config_dict["NUM_SLICES"]
     del config_dict["NUM_SLICES"]
 
+    with TaskGroup(group_id=f"generate_maxtext_run_name_{idx}") as tg:
+      run_name = maxtext.generate_maxtext_run_name(
+          run_name_prefix, curr_num_slices, tpu_version.value, tpu_cores, idx
+      )
+      tb_file_location = maxtext.generate_tb_file_location(
+          run_name, base_output_directory
+      )
+
```

Comment:
Should this be covered by `run_with_name_generation`?

Reply:
Yes good catch, I changed it to use the new `run_with_name_generation`